pages/common/cpan.md

```diff
@@ -0,0 +1,20 @@
+# cpan
+
+> Query, download and build perl modules from CPAN sites.
+> More information: <https://manned.org/man/cpan>.
```

Comment:
```suggestion
> More information: <https://manned.org/cpan>.
```

Reply:
I've incorporated the suggestion, but both URLs lead to the same page.

Reply:
> I've incorporated the suggestion, but both URLs lead to the same page.

I know, but we use `manned.org` without `/man/` on all other pages, and it is shorter.